docs/content/archiving-bundles.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Archiving Bundles
+description: Archiving Bundles with Porter
+---
+
+Porter allows you to share bundles by [publishing](/distributing-bundles) them to an OCI registry. Porter also allows you to copy a bundle from one registry to another. Using these commands, bundle users have flexibility in how they leverage published bundles. What if you want to use a published bundle on a disconnected or edge network that has limited connectivity? The `porter archive` command and the `porter publish` commands allow you to take the bundle from a registry on one network, move it to the network or location, and republish it into another registry for use within that environment. The generated bundle archive contains the CNAB `bundle.json`, along with an OCI [image layout](https://github.com/opencontainers/image-spec/blob/master/image-layout.md) containing the invocation image and any images declared in the `images` section of the `bundle.json`. This enables the entire bundle to be easily moved into a private data center or across an air-gapped network, and republished within that environment.
+
+## Generating a Bundle Archive With Porter
+
+To generate the archive, you run the `porter archive` command. For example, assume you are working with a local bundle. To generate the archive, you would run the following command:
+
+```
+porter archive bundle-archive-file-name.tgz
+```
+
+NOTE: In order to generate the archive, all of the images in the bundle **must** have been published to a registry. For this reason, it is best to first `publish` your bundle to a registry.
```

Comment:
Good news on the horizon here (as you've probably seen): https://github.com/docker/cnab-to-oci/pull/76